source/localizable/addons-and-dependencies/managing-dependencies.md

```diff
@@ -109,9 +111,15 @@ All style assets added this way will be concatenated and output as `/assets/vend
 
 ### Other Assets
 
-All other assets like images or fonts can also be added via `import()`.
-By default, they
-will be copied to `dist/` as they are.
+All assets located in the `public/` folder will be copied as is to the final output directory, `dist/`.
+
+For example, a `favicon` located at `public/images/favicon.ico` will be copied to `dist/images/favicon.ico`.
+
+All third-party assets, included either manually in `vendor/` or via a package manager like Bower, must be added via `import()`.
```

Comment:
Updated phrasing to more strongly state that third-party assets need to be included via `import`, otherwise they won't make the build.